src/main/java/ru/skypro/homework/service/impl/UsersServiceImpl.java

```diff
@@ -46,17 +50,20 @@ public class UsersServiceImpl implements UsersService {
 
     private final UserRepository userRepository;
     private final UsersMapping usersMapping;
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
```suggestion
    private final PasswordEncoder bCryptPasswordEncoder;
```
инжектить надо по интерфейсу, а не реализации